lanternmq/README.md

```diff
@@ -6,4 +6,28 @@ The package includes a RabbitMQ implementation for the LanternMQ interface.
 
 The package also includes a mock implementation for the LanternMQ interface to support testing.
 
-To test the package, see the [testing instructions](test/README.md).
\ No newline at end of file
+To test the package, see the [testing instructions](test/README.md).
+
+## Updating Users for RabbitMQ
+
+The default users, their password hashes, and each user's permissions can be found in `lantern/definitions.json`.
+
+**To update users from the RabbitMQ browser interface:**
+1. Log-in as an admin at `localhost:15672`
+2. Go to the *Admin* tab
+3. Click the specific user to update
+4. Near the bottom of the page click *Update this User*
+5. Set a new password
+
+A new user can be added by clicking *Add a user* in the *Admin* tab.
+  
+**To update users from the command line:**
+1. Access the LanternMQ docker container: `command docker exec -it <container ID> /bin/bash`
+2. Update the user's password: `rabbitmqctl change_password <username> <new password>`
+
+A new user can be added by running `rabbitmqctl add_user <username> <password>`
+
+**The definitions.json file must be updated to persist these changes. To update the definitions.json file:**
+1. Get the updated JSON object by using the RabbitMQ API <br>
+  (e.g. `curl -H "Accept:application/json" -u <username>:<password> "localhost:15672/api/definitions"`)
```

Comment:
Recommend noting that the management user account credentials need to be used here. Also, can do the original definitions file replacement by running `curl -H "Accept:application/json" -u <management_username>:<management_password> "localhost:15672/api/definitions" > lanternmq/definitions.json`